flink-core/src/main/java/org/apache/flink/configuration/HistoryServerOptions.java

```diff
@@ -89,6 +89,16 @@
 			.withDescription("Enable HTTPs access to the HistoryServer web frontend. This is applicable only when the" +
 				" global SSL flag security.ssl.enabled is set to true.");
 
+	/**
+	 * Specifies whether the HistoryServer should process job archive deletion. If this option is
+	 * enabled then deleted job archives are also deleted from HistoryServer.
+	 */
+	public static final ConfigOption<Boolean> HISTORY_SERVER_CLEANUP_EXPIRED_JOBS =
+		key("historyserver.web.clean-expired-jobs")
+			.defaultValue(false)
+			.withDescription("Whether HistoryServer should cleanup jobs" +
+				" that are no longer present in historyserver.archive.fs.dir.");
```

Comment:
Refer to `HISTORY_SERVER_ARCHIVE_DIRS.key()` instead of hard-coding the key.

Reply:
Reffered, thanks.